features/matchmaking/catena-matchmaker.md

```diff
@@ -0,0 +1,130 @@
+---
+markdown:
+  toc:
+    depth: 3
+---
+
+# Catena - The Catena Matchmaker
+The Catena Matchmaker is responsible for grouping players or parties together to form games or matches.
+
+It is _not_ responsible for assigning dedicated game servers or fleet management. If you would like to learn more about how Catena handles dedicated game servers, refer to the [Match Broker](../game-servers/index.md) documentation.
```

Comment:
We *might* want to include in this note that it also doesn't handle server tickets for backfill - backfill is also handled by the match broker.